patches/net/minecraft/gametest/framework/TestCommand.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/gametest/framework/TestCommand.java
++++ b/net/minecraft/gametest/framework/TestCommand.java
+@@ -315,7 +_,7 @@
+ 
+    private static void runTest(ServerLevel p_127930_, BlockPos p_127931_, @Nullable MultipleTestTracker p_127932_) {
+       StructureBlockEntity structureblockentity = (StructureBlockEntity)p_127930_.getBlockEntity(p_127931_);
+-      String s = structureblockentity.getStructurePath();
++      String s = structureblockentity.getMetaData().isBlank() ? structureblockentity.getStructureName() : structureblockentity.getMetaData(); // Neo: use the metadata for the structure name
```

Comment:
You use structure name here but the original line used structure path, is this intentional?

Reply:
Yes, the structure path isn't the same as the test name in most scenarios of modded game tests, since empty structures are far more common.